MAVProxy/modules/mavproxy_fakegps.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+'''fake GPS input using GPS_INPUT packet'''
+
+import time
+from pymavlink import mavutil
+from MAVProxy.modules.lib import mp_module
+from MAVProxy.modules.lib import mp_settings
+
+class FakeGPSModule(mp_module.MPModule):
+    def __init__(self, mpstate):
+        super(FakeGPSModule, self).__init__(mpstate, "relay")
+        self.last_send = time.time()
+        self.FakeGPS_settings = mp_settings.MPSettings([("nsats", int, 16),
+                                                     ("lat", float, -35.363261),
```

Comment:
Indenting issues.   `flake8`s static analysis is worth keeping things flake8-clean IMO.

Reply:
fixed